packages/dht/protos/DhtRpc.proto

```diff
@@ -177,7 +177,15 @@ message RouteMessageWrapper {
 
 message RouteMessageAck {
   string requestId = 1;
-  string error = 2;
+  optional RouteMessageError error = 2;
+}
+
+enum RouteMessageError {
+  NO_TARGETS = 0;
+  DUPLICATE = 1;
+  // TODO: can this be removed? If DhtNode is already stopped the server side requests
```

Comment:
Should we decide what to do with this before we release the next version?

Reply:
Could try to remove the stopped guards, but it could lead to memory leaks etc. Might be worth a try.